internal/pkg/options/options.go

```diff
@@ -86,17 +74,30 @@ func (o *Options) Load(logger *zap.SugaredLogger, fs filesystem.Fs, flags *pflag
 	return nil
 }
 
-func (o *Options) loadEnvFiles(fs filesystem.Fs) (map[string]string, error) {
+func (o *Options) BindFlagsAndEnvs(flags *pflag.FlagSet) error {
+	if err := o.BindPFlags(flags); err != nil {
+		return err
+	}
+
+	// For each know flag (config key) -> search for ENV
+	envs := make(map[string]interface{})
+	for _, flagName := range o.AllKeys() {
+		envName := o.envNaming.Replace(flagName)
+		if v, found := o.envs.Lookup(envName); found {
+			envs[flagName] = v
+		}
+	}
+
+	// Set config, it has < priority as flag.
+	// ... so flag value is used if set, otherwise env is used.
+	return o.MergeConfigMap(envs)
```

Comment:
Hodnoty `envs` nastavime ako config - to ma prioritu nizsiu ako flags. Vid testy dalej.